### PR TITLE
Tuple fix

### PR DESCRIPTION
**Pull request by gsmj**
Removed all tuple args from functions and replaced with separate args

### CODE REVIEW

**Comment by gsmj** [comment]
Added import fix from https://github.com/pysamp/PySAMP/pull/116

**Comment by dennorske** [comment]
Why are two modules (Dialog / Menu) in the root folder?

File: dialog.py
Comment (dennorske) [inline dialog.py:1]: Odd placement

File: menu.py
Comment (dennorske) [inline menu.py:1]: Same here, odd placement

**Review by dennorske — CHANGES_REQUESTED** [review]
Make sure there are no odd file placements and/or justify the changes in your PR

File: dialog.py
Comment (gsmj) [reply dialog.py:1]: Fixed

File: menu.py
Comment (gsmj) [reply menu.py:1]: Fixed too

File: pysamp/player.py [hunk 1]
```diff
@@ -328,19 +318,14 @@ def get_ammo(self) -> int:
         """Get the amount of ammo in the player's current weapon."""
         return get_player_ammo(self.id)
 
-    def set_ammo(self, conf: Tuple[int, int]) -> bool:
+    def set_ammo(self, weaponid: int, ammo: int) -> bool:
         """Set ammo for a weapon id.
 
         :param tuple conf: A tuple containing the weapon id and ammo.
```
Comment (dennorske) [inline pysamp/player.py]: We should make sure the docstrings are modified as well. I can do this if you want, but worth keeping in mind for future changes (so we don't get drift ^^)

**Review by dennorske — APPROVED** [review]

**Comment by dennorske** [comment]
Thanks a lot! Merging.

**Pull request merged**